Corgibytes.Freshli.Lib.Test/Initializer.cs

```diff
@@ -10,17 +10,13 @@ public static void Initialize()
         {
             VerifyDiffPlex.Initialize();
 
-            VerifierSettings.ModifySerialization(settings =>
-            {
-                settings.DontScrubNumericIds();
```

Comment:
Is there no equivalent for this call?

Reply:
As far as I'm aware, it was completely removed. See: https://github.com/VerifyTests/Verify/pull/526